alpine/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.5
+ARG CLOUD_SDK_VERSION=156.0.0
+ARG SHA256SUM=f47f41e7c389301dfee4879d6eb71d36c7b06aaf43a2be49a9ea39749be22851
+ENV PATH /google-cloud-sdk/bin:$PATH
+RUN apk --no-cache add curl python py-crcmod bash libc6-compat && \
```

Comment:
also are you using `curl` only to download the file below? or is it used by something else?